src/client/components/loader-animation/loader-animation.style.css

```
@@ -0,0 +1,42 @@
+body {
+  background: #dbdfe2;
+}
+
+.loader {
+  position: absolute;
+  top: 47%;
+  left: 47%;
+}
+
+.bubble {
+  animation: expand 1s ease-in-out infinite;
+  border-radius: 10px;
+  display: inline-block;
+  transform-origin: center center;
+  margin: 0 3px;
```

and a little bit bigger margin
```suggestion
  margin: 0 5px;
```